CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Change Log - @itwin/insights-client
 
+## 0.5.4
+Tuesday, 12 December, 2023
+### Minor
+- Added `GroupCreateCopy` and `SourceGroupReference` interfaces.
+- Added `GROUPING_AND_MAPPING_BASE_PATH` in `OperationsBase.ts`.
```

Comment:
Don't need to specify where you added it. Packages will be able to import it with
```
import { GROUPING_AND_MAPPING_BASE_PATH } from "@itwin/insights-client";
```

Reply:
I have updated the changelog